downstream/modules/platform/proc-installing-the-ansible-builder-rpm.adoc

```diff
@@ -21,214 +18,46 @@ This is preferred if a Satellite exists because the {ExecEnvShort} images can us
-+
-----
-$ cat ansible.cfg
-[galaxy]
-server_list = private_hub
-
-[galaxy_server.private_hub]
-url=https://<hub_fqdn>/api/galaxy/
-token=<admin_token>
-----
-+
-. Create a __ubi.repo__ file that points to your disconnected UBI repo mirror (this could be your Satellite if the UBI content is hosted there).
-+
-This is an example output where `reposync` was used to mirror the UBI repos.
-+
-----
-$ cat ubi.repo
-[ubi-8-baseos]
-name = Red Hat Universal Base Image 8 (RPMs) - BaseOS
-baseurl = http://<ubi_mirror_fqdn>/repos/ubi-8-baseos
-enabled = 1
-gpgkey = file:///etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release
-gpgcheck = 1
-
-[ubi-8-appstream]
-name = Red Hat Universal Base Image 8 (RPMs) - AppStream
-baseurl = http://<ubi_mirror_fqdn>/repos/ubi-8-appstream
-enabled = 1
-gpgkey = file:///etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release
-gpgcheck = 1
-----
+. Install the {Builder} RPM from the {PlatformNameShort} repository.  
 
-. Add the CA certificate used to sign the {PrivateHubName} web server certificate.
+. Subscribe the RHEL system to a Satellite on the disconnected network.
 
-.. For self-signed certificates (the installer default), make a copy of the file __/etc/pulp/certs/root.crt__ from your {PrivateHubName} and name it *hub-root.crt*.
+. Attach the {PlatformNameShort} subscription and enable the AAP repository.  The repository name will either be `ansible-automation-platform-2.4-for-rhel-8-x86_64-rpms` or `ansible-automation-platform-2.4-for-rhel-9-x86_64-rpms` depending on the version of RHEL used on the underlying system.
 
-.. If you used an internal certificate authority to request and sign the {PrivateHubName} web server certificate, make a copy of that CA certificate called *hub-root.crt*.
+. Install the {Builder} RPM.  The version of the {Builder} RPM must be 3.0.0 or later in order for the examples below to work properly.
 
-. Create your python requirements.txt and ansible collection requirements.yml with the content needed for your custom {ExecEnvShort} image.
+. Install the {Builder} RPM from the {PlatformNameShort} setup bundle.  Use this method if a Satellite server is not available on your disconnected network.
 
-. Use ansible-builder to create the context directory used to build the {ExecEnvShort} image.
-+
-----
-$ ansible-builder create
-Complete! The build context can be found at: /home/cloud-user/custom-ee/context
-$ ls -1F
-ansible.cfg
-context/
-execution-environment.yml
-hub-root.crt
-pip.conf
-requirements.txt
-requirements.yml
-ubi.repo
-----
+. Unarchive the {PlatformNameShort} setup bundle.
 
-. Copy the files used to override the internet-facing defaults into the context directory.
-+
-----
-$ cp ansible.cfg hub-root.crt pip.conf ubi.repo context/
-----
-+
-. Edit the file *context/Containerfile* and add the following modifications.
+. Install the {Builder} RPM and its dependencies from the included content.
```

Comment:
the formatting is off here.  the procedure should include two options

- either "install the ansible-builder RPM from the Ansible Automation Platform repository
- or "Install the ansible-builder RPM from the Ansible Automation Platform setup bundle"

under each of these steps, there should be the sub-steps performed for installing the RPM(s) from the relevant source.